com.bitwarden.desktop.yaml

```
@@ -63,7 +63,7 @@ modules:
         dest-filename: bitwarden.sh
         commands:
           - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
-          - exec zypak-wrapper /app/Bitwarden/bitwarden --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations "$@"
+          - exec zypak-wrapper /app/Bitwarden/bitwarden "$@"
```

```suggestion
          - exec zypak-wrapper /app/Bitwarden/bitwarden --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations "$@"
```
this is not necessary coping works when running on wayland when the x11 socket is accessible